src/Compilers/Core/Portable/InternalUtilities/PlatformInformation.cs

```diff
@@ -14,7 +15,11 @@ namespace Roslyn.Utilities
     /// </summary>
     internal static class PlatformInformation
     {
+#if NETCOREAPP
+        [SupportedOSPlatformGuard("windows")]
```

Comment:
⚠️ NETCOREAPP includes netcoreapp3.1, which does not define the attribute used here. Since the attribute first shipped in net5.0, the following would be more appropriate:

```suggestion
#if NET5_0_OR_GREATER
        [SupportedOSPlatformGuard("windows")]
```